packages/frontend-v2/utils/logger.ts

```diff
@@ -0,0 +1,16 @@
+type LogType = "info" | "debug" | "warn" | "error";
+
+/**
+ * A logger utilty that encapsulates all logging logic. This will eventually have logic that
+ * handles production logging using tools like sentry.
+ */
+const log = (type: LogType, ...messages: unknown[]) => {
+  console[type](...messages);
+};
+
+export const logger = {
+  info: (...messages: unknown[]) => log("info", ...messages),
```

Comment:
hmm maybe im trolling but why is this unknow[] instead of just string? Is it just how console works? I just thought the type unknown[] doesn't really provide that much info

Reply:
haha yeap type unknown[] doesn't provide any information but the idea is that we want the logger to by able to take a comma separated list of anything in(objects, strings, number, etc) just like the console functions do. 

I could type it as any[] but unknown[] is stricter since it prevents the logger from assuming anything about the data that's coming in. And frankly i think the logger shouldn't know/care about it is getting especially if it's just passing it to the console for now

Reply:
gotcha